src/components/Config/Config.tsx

```diff
@@ -18,7 +18,10 @@ function Config() {
   const [confirmationTime, setConfirmationTime] = useState(ConfirmationTime.oneDay);
   const [submitted, setSubmitted] = useState(false);
   const [completed, setCompleted] = useState(false);
+  const [addedConnetionModalDisplay, setAddedConnetionModalDisplay] = useState(false);
   const [network, setNetwork] = useState(Network.none);
+  const localStorage = window.localStorage;
+  const redirectPage = localStorage.getItem('strategyPath');
```

Comment:
`'strategyPath'` is a magic string that you use in more than one place. That is not acceptable, because neither linting nor TypeScript will be aware if you make a typo so it's much more error prone. NEVER do it like this. If you need a string parameter then you need to abstract it away so it's in one place ONLY.